4-forms-validations/forms.html

```diff
@@ -0,0 +1,140 @@
+          <br />
+          <li>
+            <label for="last_Name"
+              ><strong>Last Name:</strong>
+              <abbr title="required" aria-label="required">*</abbr>
+              <input
+                type="text"
+                id="last_Name"
+                placeholder="Insert your last name"
+                required
+            /></label>
+          </li>
+          <br />
+          <li>
+            <label for="email"
+              ><strong>Email:</strong>
+              <abbr title="required" aria-label="required">*</abbr>
+              <input
+                type="email"
+                id="email"
+                placeholder="Insert your Email"
+                required
+            /></label>
+          </li>
+          <br />
+          <li>
+            <label for="birth_date"
+              ><strong>Write your BirthDate:</strong>
+              <abbr title="required" aria-label="required">*</abbr>
+              <input
+                type="date"
+                id="birth_date"
+                placeholder="Insert your birth_date"
+                required
+            /></label>
+          </li>
+          <br />
+          <li>
+            <label for="genre"
+              ><strong>Pick your Genre:</strong>
+              <abbr title="required" aria-label="required">*</abbr>
+              <select id="genre">
+                <option value="Male">Male</option>
+                <option value="Female">Female</option>
+                <option value="Other">Other</option>
+              </select>
+            </label>
+          </li>
+          <br />
+          <li>
+            <label for="hobbies">
+              <strong>Pick your Hobbies:</strong>
+              <abbr title="you can choose one or more" 
+                >^</abbr
+              >
+              <br />
+              <input type="radio" name="gaming" />
+              <label for="gaming">Playing Videogames</label>
+              <br />
+              <input type="radio" name="music" />
+              <label for="music">Play a instrument</label>
+              <br />
+              <input type="radio" name="cooking" />
+              <label for="cooking">Cooking for fun</label>
+              <br />
+              <input type="radio" name="origami" />
+              <label for="origami">Crafting origami or related</label>
+              <br />
+              <label for="other"><i>Another hobby that isn't listed here?</i></label>
+              <input type="text" name="other" placeholder="Which one" maxlength="20"/>
+            </label>
+          </li>
+          <br />
+          <li>
+            <label for="profile_photo"
+              ><strong>Select a Profile Picture:</strong>
+              <abbr title="required" aria-label="required">*</abbr>
+            </label>
+            <br />
+            <input type="file" id="profile_photo" />
+          </li>
+        </ol>
+        <button type="submit">Send data</button>
```

Comment:
```suggestion
        <button>Send data</button>
```
Submit is the default behavior so you don't need to specify it.